0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -38,12 +40,15 @@ This spec depends on the [ILP spec](../0003-interledger-protocol/).
 | [**incoming_fulfill**](#event-_fulfill) | <code>( transfer:[IncomingTransfer](#incomingtransfer), fulfillment:String ) ⇒</code> |
 | [**incoming_reject**](#event-_reject) | <code>( transfer:[IncomingTransfer](#incomingtransfer), rejectionReason:[RejectionMessage](#class-rejectionmessage) ) ⇒</code> |
 | [**incoming_cancel**](#event-_cancel) | <code>( transfer:[IncomingTransfer](#incomingtransfer), cancellationReason:[RejectionMessage](#class-rejectionmessage) ) ⇒</code> |
-| [**incoming_message**](#event-_message) | <code>( message:[IncomingMessage](#incomingmessage) ) ⇒</code> |
+| [**incoming_request**](#event-_request) | <code>( message:[Message](#message) ) ⇒</code> |
+| [**incoming_response**](#event-_response) | <code>( message:[Message](#message) ) ⇒</code> |
 | [**outgoing_transfer**](#event-_transfer) | <code>( transfer:[outgoingTransfer](#outgoingtransfer) ) ⇒</code> |
 | [**outgoing_prepare**](#event-_prepare) | <code>( transfer:[outgoingTransfer](#outgoingtransfer) ) ⇒</code> |
 | [**outgoing_fulfill**](#event-_fulfill) | <code>( transfer:[outgoingTransfer](#outgoingtransfer), fulfillment:String ) ⇒</code> |
 | [**outgoing_reject**](#event-_reject) | <code>( transfer:[outgoingTransfer](#outgoingtransfer), rejectionReason:[RejectionMessage](#class-rejectionmessage) ) ⇒</code> |
 | [**outgoing_cancel**](#event-_cancel) | <code>( transfer:[outgoingTransfer](#outgoingtransfer), cancellationReason:[RejectionMessage](#class-rejectionmessage) ) ⇒</code> |
+| [**outgoing_request**](#event-_request) | <code>( message:[Message](#message) ) ⇒</code> |
+| [**outgoing_response**](#event-_response) | <code>( message:[Message](#message) ) ⇒</code> |
 | [**info_change**](#event-info_change) | <code>( info:[LedgerInfo](#class-ledgerinfo) ) ⇒</code> |
 
 ###### Errors
```

Comment:
I don't want to edit Stefan's PR, I just want to rebase and merge it. We can do a follow-up to add this if it's missing.